setup.py

```diff
@@ -1,26 +1,22 @@
 # coding: utf-8
 
-"""
-    Talon.One API
+from setuptools import setup, find_packages
+import pathlib
 
-    Use the Talon.One API to integrate with your application and to manage applications and campaigns:  - Use the operations in the [Integration API section](#integration-api) are used to integrate with our platform - Use the operation in the [Management API section](#management-api) to manage applications and campaigns.  ## Determining the base URL of the endpoints  The API is available at the same hostname as your Campaign Manager deployment. For example, if you access the Campaign Manager at `https://yourbaseurl.talon.one/`, the URL for the [updateCustomerSessionV2](https://docs.talon.one/integration-api#operation/updateCustomerSessionV2) endpoint is `https://yourbaseurl.talon.one/v2/customer_sessions/{Id}`   # noqa: E501
-
-    The version of the OpenAPI document: 
-    Generated by: https://openapi-generator.tech
-"""
-
-
-from setuptools import setup, find_packages  # noqa: H301
-
-NAME = "talon_one"
```

Comment:
Changes like this are really hard to maintain. This is due to the nature of most (if not all) files in this repo are generated through templates. Also. this name is _probably_ used for both the python package as well as the Github URL of the repo 😭 I really hope I am wrong, it's just experience with some of these language generators (though each of them is maintained by different people and can have/support different features 🤞 )

We might be able to control/override some of these (also maybe `keywords` below) via the [config file](https://github.com/talon-one/talon-service/blob/4a1c2964abae623f94b0b3718ddc71633a09e091/sdks/bundle/codegen/python/config.json#L1) (not everything is available, unfortunately)

I have pulled the supported config options to save you some time:
```
CONFIG OPTIONS

	generateSourceCodeOnly
	    Specifies that only a library source code is to be generated. (Default: false)

	hideGenerationTimestamp
	    Hides the generation timestamp when files are generated. (Default: true)

	library
	    library template (sub-template) to use: asyncio, tornado, urllib3 (Default: urllib3)

	packageName
	    python package name (convention: snake_case). (Default: openapi_client)

	packageUrl
	    python package URL.

	packageVersion
	    python package version. (Default: 1.0.0)

	projectName
	    python project name in setup.py (e.g. petstore-api).

	sortParamsByRequiredFlag
	    Sort method arguments to place required parameters before optional parameters. (Default: true)

	useNose
	    use the nose test framework (Default: false)
```

Similar to my last comment - [this](https://github.com/OpenAPITools/openapi-generator/blob/v4.3.1/modules/openapi-generator/src/main/resources/python/setup.mustache) is the mustache template file you might need to override (to add the `long_description` part) and inspect for template config options and names to override, if you deem it's worth it at the end :)